gitlab3/__init__.py

```diff
@@ -352,39 +349,38 @@ def _convert_dates(self, data):
     def _convert_gitlab_date(self, datetime_str):
         """Convert GitLab datetime string to datetime object"""
         fmt = '%Y-%m-%dT%H:%M:%S'
-
-        if datetime_str.endswith('Z'):
-            datetime_str = datetime_str[:-1]
-            offset = '+00:00' # UTC
-        elif re.search(r'\+[0-9]{2}:[0-9]{2}$', datetime_str):
-            offset = datetime_str[-6:]
-            datetime_str = datetime_str[:-6]
+        offset = None
+        if ':' in datetime_str:
+             if datetime_str.endswith('Z'):
+                 datetime_str = datetime_str[:-1]
+                 offset = '+00:00' # UTC
+             elif re.search(r'\+[0-9]{2}:[0-9]{2}$', datetime_str):
+                 offset = datetime_str[-6:]
+                 datetime_str = datetime_str[:-6]
+             else:
+                 offset = None
+
+             if re.search(r'\.[0-9]{3,6}$', datetime_str):
+                 fmt += '.%f'  # microseconds are included
         else:
-            offset = None
-
-        if re.search(r'\.[0-9]{3,6}$', datetime_str):
-            fmt += '.%f'  # microseconds are included
-
+             fmt = '%Y-%m-%d'
         dt = datetime.strptime(datetime_str, fmt)
-        if offset is None:
+        if not offset:
             return dt
-
         class GitLabTzInfo(tzinfo):
-            def __init__(self, utcoffset=0, tzname=None):
-                super(GitLabTzInfo, self).__init__()
+            def __init__(self, utcoffset):
                 self.utcoffset_val = timedelta(minutes=utcoffset)
-                self._tzname = tzname
             def utcoffset(self, dt):
                 return self.utcoffset_val
-            def dst(self, dt=None):
-                # DST not in effect (fixed offset class)
-                return timedelta(0)
-            def tzname(self, dt=None):
-                return self._tzname
-
-        hours = int(offset[:3])
+            def dst(self):
+                return None
+        sign = offset[0]
+        hours = int(offset[1:3])
         minutes = int(offset[-2:])
-        return dt.replace(tzinfo=GitLabTzInfo(hours * 60 + minutes, offset))
```

Comment:
(original date example 2017-07-08T09:30:00.515Z) because know it's possible to get only  2017-07-08 so I have fixed this error with changed format if doesn't contains " : " .